diff --git a/node_modules/@storybook/addon-cssresources/dist/index.js b/node_modules/@storybook/addon-cssresources/dist/index.js
index 9e6ed07..4875567 100644
--- a/node_modules/@storybook/addon-cssresources/dist/index.js
+++ b/node_modules/@storybook/addon-cssresources/dist/index.js
@@ -35,8 +35,8 @@ var getElement = function (id, code) {
 };
 var updateElement = function (id, code, value) {
     var _a = getElement(id, code), element = _a.element, created = _a.created;
-    element.querySelectorAll('link').forEach(function (child) { return changeMediaAttribute(child, value); });
-    element.querySelectorAll('style').forEach(function (child) { return changeMediaAttribute(child, value); });
+    Array.prototype.slice.call(element.querySelectorAll('link')).forEach(function (child) { return changeMediaAttribute(child, value); });
+    Array.prototype.slice.call(element.querySelectorAll('style')).forEach(function (child) { return changeMediaAttribute(child, value); });
     if (created) {
         global_1.document.body.appendChild(element);
     }
