diff --git a/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js b/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js
index f8e9df0..cff9477 100644
--- a/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js
+++ b/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js
@@ -199,9 +199,8 @@ function (_Component) {
         key: "filter",
         active: !!filter,
         title: "Color Blindness Emulation"
-      }, _react.default.createElement(_components.Icons, {
-        icon: "mirror"
-      })));
+      }, "Color Filter"
+      ));
     }
   }]);
 
