diff --git a/node_modules/@storybook/addon-viewport/dist/Tool.js b/node_modules/@storybook/addon-viewport/dist/Tool.js
index 7482f94..3ea0ba6 100644
--- a/node_modules/@storybook/addon-viewport/dist/Tool.js
+++ b/node_modules/@storybook/addon-viewport/dist/Tool.js
@@ -271,9 +271,8 @@ function (_Component) {
         key: "viewport",
         title: "Change the size of the preview",
         active: !!item
-      }, _react.default.createElement(_components.Icons, {
-        icon: "grow"
-      })))) : null;
+      }, "Viewport"
+      ))) : null;
     }
   }]);
 
