diff --git a/node_modules/@storybook/addon-viewport/dist/Tool.js b/node_modules/@storybook/addon-viewport/dist/Tool.js
index f4f9bc5..7be28de 100644
--- a/node_modules/@storybook/addon-viewport/dist/Tool.js
+++ b/node_modules/@storybook/addon-viewport/dist/Tool.js
@@ -129,10 +129,10 @@ var createItem = (0, _memoizerific["default"])(1000)(function (id, name, value,
 var flip = function flip(_ref3) {
   var width = _ref3.width,
       height = _ref3.height;
-  return {
+  return Object.assign({}, _ref3, {
     height: width,
     width: height
-  };
+  });
 };
 
 var deprecatedViewportString = (0, _utilDeprecate["default"])(function () {
