diff --git a/node_modules/@storybook/ui/dist/components/preview/preview.js b/node_modules/@storybook/ui/dist/components/preview/preview.js
index 9770341..9b961db 100644
--- a/node_modules/@storybook/ui/dist/components/preview/preview.js
+++ b/node_modules/@storybook/ui/dist/components/preview/preview.js
@@ -242,7 +242,7 @@ var getTools = (0, _memoizerific.default)(10)(function (getElements, panels, act
             return setGrid(!grid);
           },
           title: "Toggle background grid"
-        }, _ref9);
+        }, "Grid");
       });
     }
   }]);
