diff --git a/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js b/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js
index c654893..e27706a 100644
--- a/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js
+++ b/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js
@@ -214,9 +214,7 @@ function (_Component) {
         key: "filter",
         active: !!active,
         title: "Color Blindness Emulation"
-      }, _react["default"].createElement(_components.Icons, {
-        icon: "mirror"
-      })));
+      }, "Color Filter"));
     }
   }]);
 
