diff --git a/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js b/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js
index 96b3a86..2480a37 100644
--- a/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js
+++ b/node_modules/@storybook/addon-a11y/dist/components/ColorBlindness.js
@@ -222,9 +222,8 @@ function (_Component) {
         key: "filter",
         active: !!filter,
         title: "Color Blindness Emulation"
-      }, _react["default"].createElement(_components.Icons, {
-        icon: "mirror"
-      })));
+      }, "Color Filter"
+      ));
     }
   }]);
 
