diff --git a/node_modules/twing/lib/token-parser/spaceless.js b/node_modules/twing/lib/token-parser/spaceless.js
index 3c09e1f..269aceb 100644
--- a/node_modules/twing/lib/token-parser/spaceless.js
+++ b/node_modules/twing/lib/token-parser/spaceless.js
@@ -16,7 +16,6 @@ const token_1 = require("../token");
 const spaceless_1 = require("../node/spaceless");
 class TwingTokenParserSpaceless extends token_parser_1.TwingTokenParser {
     parse(token) {
-        console.error('The "spaceless" tag is deprecated since Twig 2.7, use the "spaceless" filter instead.');
         let lineno = token.getLine();
         let columnno = token.getColumn();
         this.parser.getStream().expect(token_1.TwingToken.BLOCK_END_TYPE);
